dbt_subprojects/daily_spellbook/models/_metrics/fees/chains/solana/metrics_solana_gas_fees_daily.sql

```diff
@@ -13,36 +15,42 @@ with fees as (
     select
         blockchain
         , block_date
+        , tx_from as address
         , sum(tx_fee_usd) as gas_fees_usd
     from
         {{ source('gas', 'fees') }}
-    where blockchain = 'solana'
-    {% if is_incremental() %}
-    and
-        {{ incremental_predicate('block_date') }}
-    {% endif %}
+    where
+        blockchain = '{{ blockchain }}'
+        {% if is_incremental() or true %}
+        and {{ incremental_predicate('block_date') }}
+        {% endif %}
     group by
         blockchain
         , block_date
+        , tx_from
 ), solana_vote_fees as (
     -- solana vote fees are stored in a different spell due to data volume & lack of value-add for materializing the fee breakdown
     select
         blockchain
         , block_date
+        , tx_from as address
         , sum(tx_fee_usd) as gas_fees_usd
     from
         {{ source('gas_solana', 'vote_fees') }}
-    {% if is_incremental() %}
     where
-        {{ incremental_predicate('block_date') }}
-    {% endif %}
+        1 = 1
+        {% if is_incremental() or true %}
+        and {{ incremental_predicate('block_date') }}
+        {% endif %}
     group by
         blockchain
         , block_date
+        , tx_from
 ), combined_fees as (
     select
         fees.blockchain
         , fees.block_date
+        , fees.address
         , fees.gas_fees_usd + coalesce(solana_vote_fees.gas_fees_usd, 0) as gas_fees_usd
     from
         fees
```

Comment:
doubt this join approach will work now with address introduced, will need to validate this and potentially do full outer join